### PR TITLE
PNW language support, and other fixes

### PR DESCRIPTION
**Pull request by aaronbell**
This update adds support for many PNW languages including the Salishan
language group and other coastal scripts. 

This is an update to Cascadia Code that adds support for the Salishan
language group and other coastal scripts, including:

* ʔayʔaǰuθəm/Saɬuɬtxʷ (Comox, Homalco, Klahoose / ƛohos, Sliammon / ɬəʔamɛn)
* Sechelt / she shashishalhem
* Squamish / Sḵwx̱wú7mesh sníchim
* Hul’q’umi’num’
* Hǝn̓q̓ǝmin̓ǝm
* Halq’eméylem
* Nooksack / Lhéchalosem
* SENĆOŦEN (Saanich, Semiahmoo, Sooke, Songish, Lummi)
* Lushootseed / Dəxʷləšucid (many Puget Sound nations)
* Quileute
* Klallam / Nǝxʷsƛ̓ay̓ǝmúcǝn
* Quinault
* Haida (Hlg̱aagilda X̱aayda Kil / X̱aad Kil / K’iis Xaat’aagang / Kasaan)
* Nuxalk / Bella Coola
* diiɁdiitidq
* c̓išaaʔatḥ / Tsechaht
* Nuu-chah-nulth / Nuučaan̓ułʔatḥ
* Ehattesaht Nuchatlaht
* Kwak̓wala
* Secwepemctsín / Shuswap
* St̓át̓imcets / Ucwalmícwts (Lillooet)
* Nłeʔkepmxcin / Thompson River
* Coeur d'Alene
* Nxaʔamxčín
* Syilx / Colville-Okanagan
* Upper Chehalis
* Lower Chehalis
* Salish (Kalispel / Spokane, etc.).

Other updates:
- Build script better streamlined so that the static instances can be
  built from the same UFO prepped for variable font build
- Remastered mark positioning for glyphs with existing diacritics for
  improved design for letters with multiple diacritics above.
- Fixed RVRN substitutions which were previously non-functional at the
  heavier weights. 
- Resolved other minor hinting issues for better consistency. 
- updated FONTLOG and glyphset image

## Validation Steps Performed
Contacted language experts to verify language support. 
Added hinting, and reviewed.

## New characters added:
Screen Shot 2020-09-14 at 2 36 43 PM

Corrected RVRN substitution rendering at bold weights:
Screen Shot 2020-09-14 at 2 38 05 PM

### CODE REVIEW

**Comment by aaronbell** [comment]
(may I also note that the inconsistent rendering of the various language names above is a big reason for this 😄 )

**Comment by DHowett** [comment]
You warned me that you'd drop this, and I didn't listen well enough, so I 100% deserve having to review gigantic PR :smile:

**Comment by DHowett** [comment]
(Excited! Thanks! Does this include the hyphen_less_less change from #352?)

**Comment by aaronbell** [comment]
@DHowett Yes. I'd suggest merging the other one first to reduce the risk of merge issues. :)

**Comment by DHowett** [comment]
K. We may end up with a conflict in build.py; I'll let you know in a sec.

**Comment by DHowett** [comment]
Conflict resolved. Giving this a smoketest, buzzing over a couple files, and we'll get it signed.

**Comment by aaronbell** [comment]
👍 If you want some language samples to look at, I suggest looking at the Salishan sections of http://www.languagegeek.com/

File: build.py [hunk 1]
```diff
@@ -92,44 +91,10 @@ def overlapFlag(varFont):
         # Set OVERLAP_SIMPLE bit for simple glyphs
         elif glyph.numberOfContours > 0:
             glyph.flags[0] |= 0x40
-    return varFont
-
-def build_font_instance(instance, *steps):
-    if os.path.exists(OUTPUT_DIR / "static") == False:
-        os.mkdir(OUTPUT_DIR / "static")
-
```
Comment (DHowett) [inline build.py:100]: Build fails w/o this -- let's move it up to `__main__`!

Comment (aaronbell) [reply build.py:100]: Oh whoopsy. Yeah, that should be kept.

Comment (aaronbell) [reply build.py:100]: I added the check back into the build script now where the static instances are built.

**Comment by DHowett** [comment]
This is lovely.

**Comment by aaronbell** [comment]
Thanks Dustin! If you have time, would you be able to push a new release tomorrow? :)

**Comment by DHowett** [comment]
Sure thing! Already ran the full pipeline to get it ready :)

**Comment by DHowett** [comment]
@aaronbell this is now out! :D

**Pull request merged**